packages/itwinui-react/src/core/Buttons/IconButton.tsx

```diff
@@ -71,7 +83,12 @@ export const IconButton = React.forwardRef((props, ref) => {
   );
 
   return label ? (
-    <Tooltip content={label} ariaStrategy='none'>
+    <Tooltip
+      content={label}
+      ariaStrategy='none'
+      placement={buttonGroupOrientation === 'vertical' ? 'right' : 'top'}
+      {...labelProps}
```

Comment:
Would it be safer to move the `{...labelProps}` at the top of the props passed to `<Tooltip>` to prevent the user from overriding these props?


```suggestion
      {...labelProps}
      content={label}
      ariaStrategy='none'
      placement={buttonGroupOrientation === 'vertical' ? 'right' : 'top'}
```

Reply:
at best, i can see something like this, since we exclude `content` and `ariaStrategy` from the types:
```suggestion
      content={label}
      ariaStrategy='none'
      {...labelProps}
      placement={buttonGroupOrientation === 'vertical' ? 'right' : 'top'}
```

Reply:
Ohh, right, but to confirm, should we disallow overriding `content` and `ariaStrategy since we exclude it in the type?

```suggestion
      placement={buttonGroupOrientation === 'vertical' ? 'right' : 'top'}
      {...labelProps}
      content={label}
      ariaStrategy='none'
```

Reply:
whoops youre right, my version has it flipped